RecoTracker/MeasurementDet/plugins/doubleMatch.icc

```diff
@@ -82,7 +82,8 @@ void TkGluedMeasurementDet::doubleMatch(const TrajectoryStateOnSurface& ts,
     if LIKELY (!emptyMono) {
       // mono does require "projection" for precise estimate
       TrajectoryStateOnSurface mts = fastProp(ts, geomDet().surface(), theMonoDet->geomDet().surface());
-      theMonoDet->simpleRecHits(mts, collector.estimator(), data, monoHits);
+      if LIKELY (mts.isValid())
```

Comment:
I used `LIKELY` just by analogy with similar calls in this class (I don't know if the use of `LIKELY` is recommended here, or in general).

Reply:
I also suspect it's not giving much of an advantage, but it should not be harmful.